sql-cli/config/astro-cli.json

```diff
@@ -4,28 +4,32 @@
       "preRelease": true,
       "sqlCliVersion": ">0.2.2"
     },
-    "1.8": {
+    "1.8.0": {
       "preRelease": false,
       "sqlCliVersion": ">0.2, <0.3"
     },
-    "1.9": {
+    "1.9.0": {
```

Comment:
Why did we add the micro digit here? It would probably be less maintenance if we could have only major + minor (no micro) in the config. My concern is if we need to update this document for every release of the Astro CLI. We can have a dedicated ticket to implement this on the Astro CLI side in case the current implementation doesn't support this.

Reply:
Yes, we need a ticket to work on matching the minor versions. At the moment the integration checks for exact matches.

Reply:
https://github.com/astronomer/astro-sdk/issues/1673